[2022-06-27 13:32:43,621] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_C scheduled__2022-05-10T00:00:00+00:00 [queued]>
[2022-06-27 13:32:43,676] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_C scheduled__2022-05-10T00:00:00+00:00 [queued]>
[2022-06-27 13:32:43,679] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:43,680] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:32:43,682] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:43,743] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_C> on 2022-05-10 00:00:00+00:00
[2022-06-27 13:32:43,771] {standard_task_runner.py:52} INFO - Started process 58073 to run task
[2022-06-27 13:32:43,804] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_C', 'scheduled__2022-05-10T00:00:00+00:00', '--job-id', '647', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpbggkkjfe', '--error-file', '/tmp/tmp6u1x09fq']
[2022-06-27 13:32:43,806] {standard_task_runner.py:80} INFO - Job 647: Subtask model_C
[2022-06-27 13:32:43,961] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_C scheduled__2022-05-10T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:32:44,100] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_C
AIRFLOW_CTX_EXECUTION_DATE=2022-05-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-10T00:00:00+00:00
[2022-06-27 13:32:44,102] {python.py:173} INFO - Done. Returned value was: 3
[2022-06-27 13:32:44,169] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_C, execution_date=20220510T000000, start_date=20220627T133243, end_date=20220627T133244
[2022-06-27 13:32:44,239] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:32:44,361] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
